qa/workunits/rbd/cli_generic.sh

```diff
@@ -1129,6 +1129,9 @@ test_trash_purge_schedule() {
     rbd pool init rbd2
     rbd namespace create rbd2/ns1
 
+    test "$(ceph rbd trash purge schedule list)" = "{}"
+    ceph rbd trash purge schedule status | grep '"scheduled": \[\]'
```

Comment:
nit: could use `fgrep`, but I am fine with this version too.

Reply:
`fgrep` isn't particularly widespread, but I see it's already being used in some rbd workunits.  Updated.